### PR TITLE
Overview of the project

### PR DESCRIPTION
**Pull request by BucurEva87**
In this project we achieved the following:

- Created an Item.rb file with all its associated properties and methods
- Created a book and label feature for adding books and labels associated with them
- Created a game and author feature for adding games and authors associated with them
- Created a music album and genre feature for adding music albums and genres associated with them
- Created a movie and source feature for adding movies and sources associated with them
- Persisted all the data and associations of the objects inside .json files
- Created tests for all the classes used in features + Item class
- Created SQL schemas for every class and association
- Recorded a presentation video of the project

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    if @authors.length.zero?
+      puts 'There are no authors in your catalog for the time being. Maybe add one...?'
+    else
+      @authors.each_with_index do |a, i|
+        puts "#{i + 1}) First name: #{a.first_name} | Last name: #{a.last_name}"
+      end
+    end
+  end
+
+  def add_game(multiplayer, last_played_at, publish_date, first_name, last_name)
```
Comment (muneebulrehman) [inline app.rb:85]: The Game should have much more information than you are covering right now. Currently, you only have an `author` for the book and not the `label` or `genre` or `source`. Kindly add these accordingly. You can refer to [this video](https://www.loom.com/share/ae09042fe10c44e9906b8628314b49f5) where I went into detail about what you should fix/change.

File: README.md [hunk 2]
```diff
@@ -200,33 +154,39 @@ Example:
...
+
+- GitHub: [@Tommy Marvel](https://github.com/Tommymarvel)
+- Twitter: [@Tomiwa Ibironke](https://twitter.com/tomiwa_ibironke)
+- LinkedIn: [Marvellous Ibironke](https://www.linkedin.com/in/marvellous-ibironke-54026b231/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
 
-## 🔭 Future Features <a name="future-features"></a>
```
Comment (muneebulrehman) [inline README.md:221]: Kindly make sure that you have added the section `Future Features`, including 1-3 potential features to be added. All main headers from this [README template](https://github.com/microverseinc/readme-template) should be present in a student's repo. The only optional section is FAQ.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

**Note**. _Kindly ignore my remarks about the `movie class` in the video.  It is a requirement only for a group of 4 students._

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (Tommymarvel) [reply app.rb:85]: hello @muneebulrehman thanks for the review, but i would like you to check this link where is the project being assigned to each student [link](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things_divided.md). Thanks

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    if @genres.length.zero?
+      puts 'There are no genres in your catalog for the time being. Maybe add one...?'
+    else
+      @genres.each_with_index do |g, i|
+        puts "#{i + 1}) Name: #{g.name}"
+      end
+    end
+  end
+
+  def add_music_album(on_spotify, publish_date, name)
```
[the remark this answers is not in the document]
Comment (Tommymarvel) [reply app.rb:112]: hello @muneebulrehman thanks for the review, but i would like you to check this link where is the project being assigned to each student [link](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things_divided.md). Thanks

File: app.rb [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    if @labels.length.zero?
+      puts 'There are no labels in your catalog for the time being. Maybe add one...?'
+    else
+      @labels.each_with_index do |l, i|
+        puts "#{i + 1}) Title: #{l.title} | Color: #{l.color}"
+      end
+    end
+  end
+
+  def add_book(publisher, cover_state, publish_date, title, color)
```
[the remark this answers is not in the document]
Comment (Tommymarvel) [reply app.rb:56]: hello @muneebulrehman thanks for the review, but i would like you to check this link where is the project being assigned to each student [link](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things_divided.md). Thanks

**Review by omarsalem7 — APPROVED** [review]
Hi team 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive README
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**